paper-toolbar.html

```diff
@@ -87,6 +87,7 @@
 `--paper-toolbar`            | Mixin applied to the toolbar | `{}`
 `--paper-toolbar-medium`     | Mixin applied to medium height toolbar | `{}`
 `--paper-toolbar-tall`       | Mixin applied to tall height toolbar | `{}`
+`--paper-toolbar-transition` | Transition applied to the `.animate` class | `height 0.18s ease-in`
```

Comment:
Is something else expected to add this class? If not, this might be a good time to document it.